### PR TITLE
Fastboot, destructuring, and ESLint

### LINKED ISSUE #4: Fastboot compatibility
This currently breaks Fastboot. I can make the changes necessary to get it to work tomorrow.

### PR DESCRIPTION
**Pull request by RobbieTheWagner**
I updated to ember-cli 2.7.0, switched from jshint to ESLint, destructured Ember, and added checks for fastboot.

Resolves #4

### CODE REVIEW

File: tests/helpers/module-for-acceptance.js [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
     },
 
     afterEach() {
-      if (options.afterEach) {
-        options.afterEach.apply(this, arguments);
-      }
-
-      destroyApp(this.application);
+      let afterEach = options.afterEach && options.afterEach.apply(this, arguments);
+      return Promise.resolve(afterEach).then(() => destroyApp(this.application));
```
Comment (bttf) [inline tests/helpers/module-for-acceptance.js:20]: why are changes to this file necessary?

Comment (RobbieTheWagner) [inline tests/helpers/module-for-acceptance.js:20]: Ember CLI changed this file. I ran `ember init` and pulled in all the necessary changes.

**Comment by RobbieTheWagner** [comment]
@bttf any issues with this PR?

**Comment by bttf** [comment]
@rwwagner90 it appears that this breaks with apps that do not have fastboot already installed

**Comment by RobbieTheWagner** [comment]
@bttf okay, I can fix that.

**Comment by RobbieTheWagner** [comment]
@bttf should be fixed now. Please give it a look

**Pull request merged**